### ISSUE TITLE
Remove deprecated t.a.app.getLogFile

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4094]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)| @thijstriemstra reported|
|-|-|
|Trac ID|trac#4094|
|Type|task|
|Created|2009-11-13 19:25:50Z|

```getLogFile``` in [twisted/application/app.py](twisted/application/app.py) is deprecated, time to get rid of it (eventually).

Attachments:

* [trac-4094.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/cba/cba0261d46db104901c232c86e3f77de6d6c3ea6/5d412cd7b14cbaf57ba0a52e44489d0b78e0b9f4.patch) (675 bytes) - added by SW on 2010-05-17 12:17:28Z - patch removing getLogFile

<details><summary>Searchable metadata</summary>

```
trac-id__4094 4094
type__task task
reporter__thijs thijs
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1258140350000000 1258140350000000
changetime__1274201004000000 1274201004000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)|@glyph removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun|
|-|-|

This function also has a unit test which needs to be deleted.  I'll delete that and apply the change.  Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [28962]) Remove twisted.applications.app.getLogFile

Author: SW, exarkun
Reviewer: exarkun
Fixes: [#4094](https://github.com/twisted/twisted/issues/4094)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
